src/shared/Microsoft.AzureRepos.Tests/AzureReposHostProviderTests.cs

```diff
@@ -570,6 +570,22 @@ public async Task AzureReposHostProvider_UnconfigureAsync_UseHttpPathSet_Removes
             Assert.Empty(context.Git.Configuration.Global);
         }
 
+        [PlatformFact(Platforms.Windows)]
+        public async Task AzureReposHostProvider_UnconfigureAsync_System_Windows_UseHttpPathSetAndManagerHelper_DoesNotRemoveEntry()
+        {
+            var context = new TestCommandContext();
+            var provider = new AzureReposHostProvider(context);
+
+            context.Git.Configuration.System[HelperKey] = new List<string> {"manager"};
+            context.Git.Configuration.System[AzDevUseHttpPathKey] = new List<string> {"true"};
+
+            await provider.UnconfigureAsync(ConfigurationTarget.System);
+
+            Assert.True(context.Git.Configuration.System.TryGetValue(AzDevUseHttpPathKey, out IList<string> actualValues));
+            Assert.Single(actualValues);
+            Assert.Equal("true", actualValues[0]);
+        }
+
```

Comment:
I'm not quite sure whether this test was added intentionally? I might be misinterpreting, but nothing in that test speaks "drop the -core suffix" to me... Likewise with `AzureReposHostProvider_UnconfigureAsync_User_Windows_UseHttpPathSetAndManagerHelper_RemovesEntry` below.

Reply:
These tests are to ensure that, on Windows, when uninstalling GCM (as installed from a standalone installer), we do NOT delete the `useHttpPath` setting that was placed in the `--system` config file by Git for Windows.

The new tests check to see if `helper = manager`, whereas the existing tests check we don't remove the `useHttpPath` setting when `helper = manager-core`.